### DISCUSSION TITLE
Budgets with a custom time range?

### DISCUSSION THREAD

**Discussion by SekoiaTree** [firefly-iii/firefly-iii#7374] [Firefly III] [answered]
Is it possible to make a budget and allocate a certain amount on a custom time range? I've only figured it out per-month, but I want to allocate a certain amount of money for lunches over the entire semester.

**Comment by JC5** [comment, marked as answer]
On the `/budget` page, change the date range at the top of the page to something custom. Then, set a budget amount in your preferred currency using the little + button. That amount will apply to the entire range you just set.

**Reply by SekoiaTree** [reply]
Thanks!

**Reply by SekoiaTree** [reply]
Seems that is still limited to a month, just a custom date. Is that a bug?
![image](https://user-images.githubusercontent.com/51149447/232588832-e7026c6c-573c-4d09-bf1f-2201b9d05966.png)

**Reply by JC5** [reply]
I see what you mean, I'll check it out!

**Reply by JC5** [reply]
This was a bug and should be fixed in the next release :+1:

**Reply by SekoiaTree** [reply]
Thank you! Has the docker image already updated, or am I just bad at upgrading my docker containers?

**Reply by JC5** [reply]
Not yet, I forgot about those completely! Should be up in a minute,

**Reply by SekoiaTree** [reply]
awesome, thanks.
![image](https://user-images.githubusercontent.com/51149447/235294150-f06b9591-bf02-496b-a1a5-721623e892ee.png)
Okay, now it's different, but it's still not the right range :laughing: 

Edit: actually, seems to be good? got it to budget correctly in the range
Edit 2: Yep, seems all good! Thank you!

**Reply by JC5** [reply]
Glad to hear it!
